.github/workflows/head-build.yml

```diff
@@ -76,3 +77,14 @@ jobs:
           mkdir -p oras-install/
           tar -zxf oras_${VERSION}_*.tar.gz -C oras-install/
           oras-install/oras copy ${{ env.FULL_IMAGE_URL }}:${{ needs.prebuild-env.outputs.branch_static_tag }} ${{ env.FULL_IMAGE_URL }}:${{ needs.prebuild-env.outputs.branch_tag }}
+
+      - name: Re-tag the static head image to ghcr.io/${{ github.repository_owner }}
```

Comment:
This workflow only covers the "head" builds and not stable releases. For your usage I suspect you should only target using the stable tagged releases. Also this will only cover getting the "rolling" tag to GHCR - and we'll also want the static tag mirrored as well.